Template/Template/Controllers\/OrderViewController.swift

```diff
@@ -4,8 +4,105 @@
 import UIKit
 
 /// экран с итоговой суммой
-class OrderViewController: UIViewController {
-//    override func viewDidLoad() {
-//        super.viewDidLoad()
-//    }
+final class OrderViewController: UIViewController {
+    // MARK: - Constants
+
+    enum Constants {
+        static let startFrameY = 191
+        static let frameYSpace = 36
+        static let rubel = "Руб"
```

Comment:
```suggestion
        static let currency = "Руб"
```
   и раз повторяется вынеси в отдельный глобальный энам